js/notify/notifyImpl.js

```diff
@@ -241,4 +246,46 @@ function Queue(emitter) {
     return modifiedEmitter;
 }
 
+/**
+ * Replace HTML tags <> from messages test to prevent
+ * HTML injection
+ *
+ * @param string
+ * @return {void | string | *}
+ */
+function replaceHTMLTags(string) {
```

Comment:
Could rename the param to `tag` perhaps?

Reply:
@KiranNiranjan Yes, please rename the parameter.

Reply:
@VikasShashidhar & @VishwasShashidhar But its a `String`, It may or may not contain HTML tags

Reply:
I see. Is there something else we can name it to? The line `string !== 'string'` is a little confusing IMO.

Reply:
Shall I rename it to `message`?

Reply:
yes please 🙂